src/sentry/static/sentry/app/components/events/interfaces/frameVariables.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+import {getMeta} from 'app/components/events/meta/metaProxy';
+import KeyValueList from 'app/components/events/interfaces/keyValueList/keyValueListV2';
+import {KeyValueListData} from 'app/components/events/interfaces/keyValueList/types';
+
+type Props = {
+  data: {[key: string]: string};
+};
+
+const FrameVariables = ({data}: Props) => {
+  // make sure that clicking on the variables does not actually do
+  // anything on the containing element.
+  const handlePreventToggling = () => (event: React.MouseEvent<HTMLTableElement>) => {
+    event.stopPropagation();
+  };
```

Comment:
We have this function in a few places. Is it worth adding a importable function for this?

Reply:
I agree with you, but I don't know if it would be a good idea to create a common function for this ... I like having the comments here and the functionName .. it helps to understand what we are doing

Reply:
:+1: